# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: xpcom/ds/StaticAtoms.py
# Commit: 00bf4fed8bcb
# Full Hash: 00bf4fed8bcb9b993f2bb1203817ea9f4211c22a
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2023-06-29 15:31:45
# Regressor Bug: 1831451
# File Overlap Count: 1
# Description:
#   Bug 1831451 - Fire a dequeue event when data is being decoded r=padenot
#   
#   This patch implements the dequeue-event dispatch [1] that decode method
#   and reset method need.
#   
# ==============================================================================

diff -r c8fdb24dd953 -r 00bf4fed8bcb xpcom/ds/StaticAtoms.py
--- a/xpcom/ds/StaticAtoms.py	Thu Jun 29 04:15:02 2023 +0000
+++ b/xpcom/ds/StaticAtoms.py	Thu Jun 29 04:15:02 2023 +0000
@@ -2429,6 +2429,8 @@
     Atom("onupdateend", "onupdateend"),
     Atom("onaddsourcebuffer", "onaddsourcebuffer"),
     Atom("onremovesourcebuffer", "onremovesourcebuffer"),
+    # WebCodecs
+    Atom("ondequeue", "ondequeue"),
     # RDF (not used by mozilla-central, but still used by comm-central)
     Atom("about", "about"),
     Atom("ID", "ID"),
